scripts/start-sparkswapd.sh

```diff
@@ -7,13 +7,72 @@
 #            as `/secure` is a volume inside of the container
 ################################################
 
-set -e
+set -ex
+
+# We set value defaults here to mimic the behaivor of docker-compose in-case
+# this script is called outside of docker-compose
+DATA_DIR=${DATA_DIR:-""}
```

Comment:
We should consider matching up these names with the actual config parameter, e.g. (if we stick with the `broker` prefix) `BROKER_DATA_DIR` instead of `DATA_DIR`

Reply:
I went ahead and removed broker prefix so we should be OK